src/utils/management/commands/test_fire_event.py

```diff
@@ -0,0 +1,84 @@
+import json
+import os
+from mock import Mock
+
+from django.core.management.base import BaseCommand
+from django.apps import apps
+from django.http import HttpRequest
+
+from events import logic as event_logic
+from journal import models
+from core import models as core_models
+
+
+def create_fake_request(journal, user):
+    request = Mock(HttpRequest)
+    request.GET = Mock()
+    request.journal = journal
+    request.user = user
+    request.site_type = journal
+    request.FILES = None
+    request.META = {}
+    request.press = journal.press
+    request.META = {'REMOTE_ADDR': '127.0.0.1'}
+    request.model_content_type = None
+
+    return request
+
+
+class Command(BaseCommand):
+    help = "Test fires an event so you can see the output."
+
+    def add_arguments(self, parser):
+        """Adds arguments to Django's management command-line parser.
+
+        :param parser: the parser to which the required arguments will be added
+        :return: None
+        """
+        parser.add_argument('journal_code')
+        parser.add_argument('user_id')
+        parser.add_argument('event_name')
+        parser.add_argument('json_path')
```

Comment:
It'd be usefull to also allow passing a JSON string as an argument for dockerised installations